package.json

```diff
@@ -7,23 +7,23 @@
 	"author": {
 		"name": "vdemedes",
 		"email": "vdemedes@gmail.com",
-		"url": "github.com/vadimdemedes"
+		"url": "https://github.com/vadimdemedes"
```

Comment:
Shouldn't it work without `https://` prefix?

Reply:
Unfortunately it does not, so this change is correct.